HOWTO/SBOM.md

```diff
@@ -0,0 +1,207 @@
+To run ORT locally, we detail the steps running ORT from source and from Docker.
+
+#### Steps From Source
+
+1. Install `oss-review-toolkit` (ORT) and `scancode`.
+2. Run the ORT analyzer (some paths may need tweaking depending on where you are):
+
+   ```bash
+   ./gradlew cli:run --args="-c .ort/config/config.yml analyze -i . -o . -f JSON --repository-configuration-file=.ort.yml"
+   ```
+   
+3. Run the ORT scanner (`scancode` is needed):
+
+   ```
+   ./gradlew cli:run --args="-c .ort/config/config.yml scan -o . -f JSON -i analyzer-result.json"
+   ```
+   
+4. Generate ORT SPDX report
+   ```
+   ./gradlew cli:run --args="report -i cli/scan-result.json -o . -f SpdxDocument -O SpdxDocument=outputFileFormats=JSON"
+   ```
+   
+5. From the Erlang/OTP repo, run the following escript to fix some known issues from the generated SPDX:
+
+   ```bash
+   .github/scripts/otp-compliance.es sbom otp-info --sbom-file ort/cli/bom.spdx.json --input-file ort/cli/scan-result.json
+   ```
+
+#### Steps From Docker
+
+1. Run the ORT analyzer (some paths may need tweaking depending on where you are) and choose an appropriate version, in this example `51.0.0`:
+   
+   ```bash
+   docker run ghcr.io/oss-review-toolkit/ort:51.0.0 -c .ort/config/config.yml analyze -i . -o . -f JSON --repository-configuration-file=.ort.yml
+   ```
+   
+2. Run the ORT scanner:
+   
+   ```bash
+   docker run ghcr.io/oss-review-toolkit/ort:51.0.0 -c .ort/config/config.yml scan -o . -f JSON -i analyzer-result.json
+   ```
+   
+3. Generate ORT SPDX report
+
+   ```bash
+   docker run ghcr.io/oss-review-toolkit/ort:51.0.0 report -i cli/scan-result.json -o . -f SpdxDocument -O SpdxDocument=outputFileFormats=JSON
+   ```
+   
+4. From the Erlang/OTP repo, run the following escript to fix some known issues from the generated SPDX,
+   and to separate Erlang/OTP applications into their own SPDX Packages:
+   
+   ```bash
+   .github/scripts/otp-compliance.es sbom otp-info --sbom-file ort/cli/bom.spdx.json --input-file ort/cli/scan-result.json
+   ```
+
+### Erlang/OTP SBOM Generation on Each Release
+
+Erlang/OTP generates and uploads to sigstore the generated SBOM via Github Actions.
+All the details can be found in `otp/.github/workflows/main.yaml`.
+
+## Testing Erlang/OTP SBOM
+
+The generated source SBOM should be verified. The current escript contains some basic
+verification capabilities and that be complemented by other tools, such as `ntia-conformance-checker`.
+
+To verify that the SBOM is correct, run the following escript after [Steps From Source] or [Steps From Docker]:
+
+```bash
+.github/scripts/otp-compliance.es sbom test-file --sbom-file otp.spdx.json
+```
+
+## Updating SPDX Packages
+
+Details for how to update Erlang/OTP applications and vendor dependencies follow.
+
+### Update Erlang/OTP Applications
+
+**NOTE**: The source SBOM that you are generating will be based on the current
+Erlang version that runs `otp-compliance.es`. This is simply because the version of
```

Comment:
Which dependencies are these? The version of all application can be found in `$ERL_TOP/otp_versions.table`.

Reply:
That's good to know, I was doing some findings considering apps everything that has `.app.src` + erts.
I found the same thing as the table, but the table makes better that some of those do not need to be install in the running system... I will see if I can quickly adapt to remove the version from querying the system.
Thanks!

Reply:
I will add the versions table as a check during the verification phase, thanks